packages/api/src/providers/fitbit.ts

```diff
@@ -511,11 +564,34 @@ export class Fitbit extends Provider implements OAuth2 {
         resp.data
       );
     } catch (error) {
-      console.log(`createSubscription for Fitbit failed. Url: ${url}, Cause: ${error}`);
+      console.log(`Failed to create a Fitbit WH subscription. Url: ${url}, Error: ${error}`);
       capture.error(error, {
-        extra: { context: `fitbit.createSubscription`, url },
+        extra: { context: `fitbit.createSubscription`, url, error },
+      });
+      throw new Error(`Failed to create a Fitbit WH subscription.`, {
+        cause: error,
```

Comment:
We have error handling code here (individual create), on the group create (createSubscriptions), and on `postAuth`. Let's pick one.

Reply:
i think ill report the individual occurrences to Sentry

Reply:
Sounds good. Let's just keep the description the same so they're grouped under the same issue at Sentry, and not duplicate the issue by also reporting upstream.